components/dashboard/src/configurations/detail/ConfigurationName.tsx

```diff
@@ -50,23 +51,35 @@ export const RepositoryNameForm: FC<Props> = ({ configuration }) => {
     );
 
     return (
-        <form onSubmit={updateName}>
+        <form onSubmit={updateName} className="border border-gray-300 dark:border-gray-700 rounded-xl p-4">
             <TextInputField
+                label="Configuration name"
                 hint={`The name can be up to ${MAX_LENGTH} characters long.`}
                 value={projectName}
                 error={nameError.message}
                 onChange={setProjectName}
                 onBlur={nameError.onBlur}
             />
-
-            <Button
-                className="mt-4"
-                htmlType="submit"
-                disabled={configuration.name === projectName}
-                loading={updateProject.isLoading}
-            >
-                Update Name
-            </Button>
+            <div className="flex flex-row items-center justify-start gap-2 w-full">
+                <LoadingButton
+                    className="mt-4"
```

Comment:
We should remove the margin on these buttons imo and add padding to the parent div - that way we don't have to have the margin matching on both buttons for it to look right.

Reply:
totally!